.test-cover.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "mode: set" > acc.out
+FAIL=0
+
+# Standard go tooling behavior is to ignore dirs with leading underscors
```

Comment:
:scream: Will fix.

(Stole this script from https://github.com/uber/go-torch/blob/master/.test-cover.sh)